rbac/apps.py

```
@@ -0,0 +1,19 @@
+"""
+from django.apps import AppConfig
+
+
+class RbacConfig(AppConfig):
+    """پیکربندی اپلیکیشن RBAC"""
+    
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'rbac'
+    verbose_name = 'مدیریت نقش‌ها و دسترسی‌ها'
+    
+    def ready(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `ready`

Add return type annotation: `None`

(ANN201)